development.rst

```diff
@@ -0,0 +1,70 @@
+Development
+===========
+
+To contribute to `pytest-html` you can use `Pipenv`_ to manage
```

Comment:
pytest-metadata

Reply:
*argh* And I was so careful going through everything looking for copy paste errors. 😛

Reply:
Done.